resources/public/pxls.js

```diff
@@ -2206,6 +2206,10 @@ window.App = (function () {
                         if (data) {
                             return $.map(self.hooks, function (hook) {
                                 const get = hook.get(data);
+                                if (!get) {
```

Comment:
If the hook result returns falsey then this short circuits, so e.g. if `pixel_count` is `0` then it doesn't show up in the shift+click result. This can also impact custom hooks that weren't expecting this behavior, e.g. "is on watchlist: false". This would fix most cases:
```suggestion
                                if (get == null) {
```

Reply:
Even better, have another function return whenever we should show it or not.

I might get to doing that tonight. For now though, I'll apply this.